ncf_inference/unittest_train_dataset.py

```diff
@@ -0,0 +1,37 @@
+import pandas
+import unittest
+import os
+import sys
+from TrainDataset import TrainDataset
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+class test_train(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.ratings = pandas.DataFrame({ 
+            'user_id': [0,0,0,1,1,1,2,2,2,3,3,3,4,4,4],
+            'item_id': [0,0,0,1,1,1,2,2,2,3,3,3,4,4,4],
+        })
+        self.all_item_ids = list(set(self.ratings['item_id']))
+
+        # Call the get_dataset function on the above data
+        self.td = TrainDataset(self.ratings, self.all_item_ids)
```

Comment:
Let's walk together through this initialization: As I am not able to follow clearly